src/p_user.c

```diff
@@ -525,6 +526,13 @@ void P_PlayerThink (player_t* player)
 	{
 	  P_UseLines (player);
 	  player->usedown = true;
+
+	  // [crispy] "use" button timer
+	  if (hud_time_use)
+	  {
+	    player->btuse = leveltime;
```

Comment:
Why not add `+ 1` here (with a comment!) instead of doing it twice in `HU_widget_build_sttime()`?

Reply:
Done in [b3edc7f](https://github.com/fabiangreffrath/woof/pull/1414/commits/b3edc7f4348d6758570b4ea33eda022a0c21d180).

Reply:
Oops, I just realized I forgot the comment. What should it be, though?

Reply:
The relevant commit in DSDA which adds the extra tic is this:

https://github.com/kraflab/dsda-doom/commit/5fe9698ada1b857c5aa13f9af3c3b20ad705a94a

However, @kraflab changed the comment a bit later, so let's just use this one:

https://github.com/kraflab/dsda-doom/commit/a91cb7e8cac4e3fdefbd4e5bb77ae375a6284b5a

Reply:
Done in 4764ea9aad67683dbc7e58b68aeb8248d85de0e7, fixed formatting in 28aa7375af4a60e9baa5e0651b1b9667382abd97.